File: ListViewPaginationAndroid/app/src/main/java/tr/com/fkocak/listviewpaginationandroid/api/interfaced/APIService.java
package tr.com.fkocak.listviewpaginationandroid.api.interfaced;

/**
 * Created by fatihkocak on 25.04.2019.
 */

import okhttp3.RequestBody;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Headers;
import retrofit2.http.POST;
import tr.com.fkocak.listviewpaginationandroid.model.ResponseData;

public interface APIService {

    @POST("/isg-katip/auth")
    @Headers({"Content-Type: application/json;charset=UTF-8"})
    Call<ResponseData> savePostWithPojoModel(@Body RequestBody type);

}

File: KotlinRetrofit/app/src/main/java/tr/com/fkocak/kotlinretrofit/api/ApiService.kt
package tr.com.fkocak.kotlinretrofit.api


import retrofit2.Call
import retrofit2.http.GET
import tr.com.fkocak.kotlinretrofit.mvp.model.responseModel.Hero

interface ApiService {
    @GET("marvel")
    fun getHeroes(): Call<List<Hero>>

}